yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
is there a reason this was added?

Reply:
I believe this line was automatically generated

Reply:
File removed. Fixed in [c08934c](https://github.com/rhdevs/rhapp/pull/567/commits/c08934cac26fe6193d81635264dd9b62290414b9)